tests/test_fortran/test.sh.in

```diff
@@ -1,24 +1,26 @@
 #!/bin/bash
 set -e
 
-export VFC_BACKENDS="libinterflop_mca.so"
-if grep "DRAGONEGG_PATH \"\"" ../../config.h > /dev/null; then
-	echo "this test is not run when using --without-dragonegg"
-	exit 0
+if [ -z "@FLANG_PATH@" ]; then
+    echo "this test is not run when not using --with-flang"
+    exit 0
 fi
 
-# Compile newton.f90 using MCA lib instrumentation
+export VFC_BACKENDS="libinterflop_mca.so"
 
+# Compile newton.f90 using MCA lib instrumentation
 verificarlo -c newton.f90 -o newton.o
-verificarlo newton.o -o newton
+verificarlo --fortran-linker newton.o -o newton
```

Comment:
Why do we need to specify --fortran-linker ? Cannot we have a language agnostic linker ? This is going to cause problems in more complex build scripts.

Reply:
We need it to know which linker to use during the link step. But I agree that is not convenient.
Another option is to create wrapper for each language: verificarlo-c, verificarlo-f and verificarlo-c++. That solves this problem and is easy to implement.

Reply:
When using gfortran with dragonegg. It was possible to link both c, c++ and fortran with clang++ as a linker.
I understand, that flang is probably adding some libraries at link time; so now we have to know what language we are linking. I wonder if this could be auto-detected ? If not we can keep `--fortran-linker` but we have to document it.

Reply:
If we compile and link in the row it's straightforward. But if we only link, it's pretty hard to auto-detect the source language from the object files. For some of them, you can retrieve in the DWARF info which compiler was used, for of them you cannot. I made the test with a simple c++ file and the DWARF was empty. Maybe I forgot some flags. If you have c and fortran mixed you can search which .o file has the main function. But things get worst when you want to create a library. Actually, I think that creating one wrapper for each language makes things easier, especially for compiling libraries. And it solves the problem with c++ because for now we have to manually add the `-libstd++` flag to compile c++ files.

Reply:
Ok agreed; there is no easy way to auto detect from object file.

Wrappers for each language could be nice. To avoid code duplication, I would recommend keeping verificarlo.in.in; and having additional light wrappers that call verificarlo.in.in with --fortran-linker or --c++-linker. This way, we can use the wrappers but still keep backwards compatibility and we avoid duplication. What do you think ?

Reply:
Yes, I had exactly the same idea :).